websites/B/BigBlueButton/dist/metadata.json

```diff
@@ -8,7 +8,7 @@
         "demo.bigbluebutton.org",
         "demo2.bigbluebutton.org"
     ],
-    "regExp": "\\/(b|gl)\\/[\\w]{3}-[\\d\\w]{3}-[\\d\\w]{3}|html5client\\/join\\?|client\\/guest-wait.html",
+    "regExp": "[/](b|gl)[/][\\w]{3}-[\\d\\w]{3}-[\\d\\w]{3}|html5client[/]join[?]|client[/]guest-wait[.]html",
```

Comment:
I am pretty sure that in the js varient \\d is just \d same with \\w -> \w.

Reply:
The backslashes in \d and \w have to be escaped to be inside a JSON file. \d is not an escaped symbol for JSON, but for regex, where it already has only one backslash. Not sure if I understand  what you mean correctly.

Reply:
ah aight, did you test it if it still works then?

Reply:
Yes, my last commit changing backslashes to [] still works.